stl/inc/xutility

```diff
@@ -376,17 +375,16 @@ struct _Ref_fn { // pass function object by value as a reference
 };
 
 template <class _Fn>
-_INLINE_VAR constexpr bool _Pass_functor_by_value_v = conjunction_v<bool_constant<sizeof(_Fn) <= sizeof(void*)>,
+_INLINE_VAR constexpr bool _Pass_functor_by_value = conjunction_v<bool_constant<sizeof(_Fn) <= sizeof(void*)>,

```

Comment:
Given that this is used nowhere else, should this become a constexpr variable inside the `_Pass_fn` function ?